nextflow_schema.json

```
@@ -706,6 +706,148 @@
+                    "help_text": "Specify the length from the end of the read that mapDamage should rescale at both ends.\n\n> Modifies the `--seq-length` parameter of mapDamage2.",
+                    "fa_icon": "fas fa-ruler-horizontal"
+                },
+                "damage_manipulation_rescale_length_5p": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "Length of read for mapDamage2 to rescale from 5p end. Only used if not 0 otherwise `--rescale_seqlength` used.",
+                    "help_text": "Specify the length from the end of the read that mapDamage should rescale. Overrides `--rescale_seqlength`.\n\n> Modifies the `--rescale-length-5p` parameter of mapDamage2.",
+                    "fa_icon": "fas fa-balance-scale-right"
+                },
+                "damage_manipulation_rescale_length_3p": {
+                    "type": "integer",
+                    "default": 0,
+                    "description": "Length of read for mapDamage2 to rescale from 3p end. Only used if not 0 otherwise `--rescale_seqlength` used.",
+                    "help_text": "Specify the length from the end of the read that mapDamage should rescale. Overrides `--rescale_seqlength`.\n\n> Modifies the `--rescale-length-3p` parameter of mapDamage2.",
+                    "fa_icon": "fas fa-balance-scale-left"
+                },
+                "run_pmd_filtering": {
+                    "type": "boolean",
+                    "description": "Turn on PMDtools filtering.",
+                    "help_text": "Specifies to run PMDTools for damage based read filtering in sequencing libraries. Turned off by default.",
+                    "fa_icon": "fas fa-power-off"
+                },
+                "damage_manipulation_pmdtools_threshold": {
+                    "type": "integer",
+                    "default": 3,
+                    "fa_icon": "far fa-chart-bar",
+                    "description": "Specify PMDScore threshold for PMDTools.",
+                    "help_text": "Specifies the PMDScore threshold to use in the pipeline when filtering BAM files for DNA damage. Only reads which surpass this damage score are considered for downstream DNA analysis. By default set to `3` if not set specifically by the user.\n\n> Modifies PMDTools parameter: `--threshold`"
+                },
+                "damage_manipulation_pmdtools_reference_mask": {
+                    "type": "string",
+                    "fa_icon": "fas fa-mask",
+                    "help_text": "Activates masking of the reference fasta prior to running pmdtools. Positions that are in the provided bedfile will be replaced by Ns in the reference genome. \nThis can alleviate reference bias when running PMD filtering on capture data, where you might not want the allele of a transition SNP to be counted as damage. Masking of the reference is done using `bedtools maskfasta`.",
```

```suggestion
                    "help_text": "Supplying a bedfile to this parameter activates masking of the reference fasta at the contained sites prior to running PMDtools. Positions that are in the provided bedfile will be replaced by Ns in the reference genome. \nThis can alleviate reference bias when running PMD filtering on capture data, where you might not want the allele of a transition SNP to be counted as damage. Masking of the reference is done using `bedtools maskfasta`.",
```


Ensure you've written `PMDTools` consistently everywhere